source/views/components/nav/Navbar.svelte

```diff
@@ -0,0 +1,47 @@
+<script lang="ts">
+  import type { Tab } from "./tab";
+
+  let {
+    Tabs = $bindable([]),
+    current = $bindable(""),
+  }: { Tabs: Tab[]; current: string } = $props();
+</script>
+
+<div class="tab">
+  {#each Tabs as tab}
+    <button
+      class:selected={current === tab.label}
+      class="nav-button"
+      onclick={() => {
+        current = tab.label;
+      }}><span class="text">{tab.title}</span></button
+    >
+  {/each}
+</div>
+
+<style>
+  .tab {
+    overflow: hidden;
+    background-color: rgb(50, 53, 59);
+  }
+
+  .nav-button {
```

Comment:
er der overhovedet brug for klassen nav-button hvis bare button nedenunder virker fint?

Reply:
Levn fra debugging. Fikses